packages/design-system/src/components/Loading/Loading.types.ts

```diff
@@ -0,0 +1,21 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+export interface LoadingProps extends React.HTMLAttributes<any> {
```

Comment:
lint warning here ... but also you aren't using any of the html props anyway so I don't think you even need to extend this

Reply:
Hm, I added this to address an error in `dashboards` when I passed the prop [here](https://github.com/Recidiviz/recidiviz-dashboards/pull/2980/files#diff-01161f71372a489d1ac154b7536f7eadd1d000630f0531880101b57310a25423R156). Error said something like `Type '{ showMessage: boolean; }' is not assignable to type 'IntrinsicAttributes & Props'`

Reply:
I think just passing the props type to `React.FC` should address that, without the need for the HTMLAttributes interface

Reply:
TIL! Would we need to specify a HTML attributes super if we wanted to be able props for a specific tag type i.e. `<button/>`, or does FC cover all possible tag props?

Reply:
Retried it without the extends and it worked! removed

Reply:
@ohaibbq not sure I am understanding your question right but we have done stuff like that before, e.g. [here](https://github.com/Recidiviz/web-libraries/blob/main/packages/design-system/src/components/Button/Button.types.ts#L24) ... FC doesn't assume anything HTML-specific out of the box

Reply:
Ah, gotcha, I had interpreted your original comment as if `React.FC` _**did**_  include HTML-specific props out of the box. That specific button implementation is what I had in mind.